packages/vue-to-dsl/src/parsers/templateParser.ts

```
@@ -0,0 +1,238 @@
+    const directiveName = prop.name
+    switch (directiveName) {
+      case 'if':
+        schema.condition = prop.exp ? prop.exp.content : 'true'
+        break
+      case 'for':
+        if (prop.exp) {
+          const exp = prop.exp.content || ''
+          // Extract the iterable expression in `v-for="(item, i) in/of list"`
+          // Prefer the part after the last occurrence of ` in ` or ` of `.
+          const match =
+            exp.match(/^[^]*?(?:\)|\S)\s+(?:in|of)\s+([^]+)$/) || exp.match(/^(?:[^]+?)\s+(?:in|of)\s+([^]+)$/)
```

Biome (2.1.2)

[error] The regular expression includes this negated empty character class.

Negated empty character classes match anything.
If you want to match against [, escape it \[.
Otherwise, remove the character class or fill it.

(lint/correctness/noEmptyCharacterClassInRegex)